docs/specifications/tests/Zone-TP/zone09.md

```diff
@@ -1,47 +1,276 @@
 
-It is therefore necessary for any domain to publish an MX record.
-However, SMTP can make a delivery without the MX, using the A or
-AAAA record as specified in section 5.1 of
-[RFC 5321](https://tools.ietf.org/html/rfc5321#section-5.1).
+Even if not mentioned in [RFC 2142], there are some exception to the
+rule above.
 
-### Inputs
+Zones in the .ARPA tree are meant for infrastructural identifier,
+and email domains are not expected for those ([RFC 3172]). This means
+that the well known mailbox is not expected for reverse zones
+(in-addr.arpa, ip6.arpa). Those are excluded from the requirement in
+this test case.
 
-The domain name to be tested.
+The root zone cannot be a mail domain since the mail domain is
+the part to the left of the trailing dot, e.g. "example.com" is the
+the maildomain for tha apex of "example.com.". The root zone is
+excluded from the requirement.
 
-### Ordered description of steps to be taken to execute the test case
+Top-level domains (TLDs) can technically function as mail domains
+([RCF 5321][RFC 5321#section-2.3.5], section 2.3.5) but they rarely
+have that function and are probably not meant to be included in the
+specification in [RFC 2142]. On the contrary,
+[Internet Architecture Board] concludes in a report
+"[Dotless Domains Considered Harmful][IAB Statement]" that TLD names
+should not be mail domains. In this test case TLDs are not only
+excluded from the requirement of being an mail domain, if found to
+be, a warning will be generated that points that out.
 
-1. Query the MX record from a delegated name server for the domain.
-2. If the answer from step 1 is not authoritative, iterate step 1 until there is an authoritative answer.
-3. If there are MX records present in the answer, this part of the test
-   case succeeds.
-4. If there are no MX records present in the answer from the MX query,
-   query for the A and AAAA records for the domain.
-5. If there is any A och AAAA record in the answers from the queries in
-   step 4, this test case succeeds.
+It is therefore expected for every domain (zone), excluding those
+excepted above, to publish an MX record in apex of the zone (i.e.
+in the same position as the SOA record). If MX is not present, SMTP
+*can* deliver email using an address record (A or AAAA) as specified
+in [RFC 5321][RFC 5321#section-5.1], section 5.1, but that possibility
+is not in common use. This test case only checks for MX record since
+that is the absolutely dominating way to configure support for email.
+If an address record is found in apex it is more often used for web
+service rather than mail service.
 
-### Outcome(s)
+[RFC 7505] standardizes "Null MX" which in means that there is no
+mail service for the domain. If a "Null MX" is found, it is considered
+to be the same as there is no MX.
 
-If there is no target host (MX, A or AAAA record) to deliver e-mail for the
-domain name, this test case fails.
+In this test case, the follwoing zone types are excepted from the
```

Comment:
"follwoing" -> "following"

"excepted" is very close to "expected", would "excluded" work instead here?